### PR TITLE
History downloader import

### PR DESCRIPTION
**Pull request by CremaLuca**
## Premise
Probably should have done this from the very beginning.

## Changes
+ added `download` directory.
+ added `yahoo_downloader`, the old yf_downloader from history-importer.
+ added `alphavantage_downloader` the old av_downloader but improved a bit.

As discussed [here](https://trello.com/c/lGkfXvg4) the two classes act as the adapter and perform the two functions of downloading and formatting/standardizing the data.

~ `config.get_config()` is now static and doesn't need class initialization.

I'll complete the TODO specifications as you read the code now.

### CODE REVIEW

File: download/yahoo_downloader.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import json
+import yfinance as yf
+import importer.json_key_handler as json_kh
+
+META_PROVIDER_VALUE = "yahoo finance"
+
+
+class YahooDownloader(TimeseriesDownloader):
+    '''
+    TODO: class specifications
```
Comment (ricdezen) [inline download/yahoo_downloader.py]: Beep Beep

Comment (CremaLuca) [reply download/yahoo_downloader.py]: > I'll complete the TODO specifications as you read the code now.

.* cough * * cough *

**Review by ricdezen — APPROVED** [review]
I suggest adding a method to both downloaders, something like:
`def get_most_recent(symbol, interval)` which can just be a call to `download_between_dates` with `datetime.now()` as end_date.
I don't see anything out of the ordinary except for missing class specs for both downloaders.

**Comment by CremaLuca** [comment]
I really don't think that such `get_most_recent(symbol, interval)` method could fit inside these two classes, it could be easily done by a generic class that gets a `TimeseriesDownloader` as parameter and implements the `get_most_recent(symbol, interval)` by calling `download_between_dates` with `datetime.now()` of the parameter's downloader class.

File: download/yahoo_downloader.py (hunk 1 above)
Comment (ricdezen) [reply download/yahoo_downloader.py]: Beep

**Review by ricdezen — APPROVED** [review]

**Pull request merged**